src/main/java/tsunami/security/scanner/utilities/KubeJavaClientUtil.java

```diff
@@ -54,7 +59,9 @@ private KubeJavaClientUtil() {}
           V1PersistentVolumeClaim.class,
           v1Pvc -> createPvc((V1PersistentVolumeClaim) v1Pvc),
           V1Service.class,
-          v1Service -> createService((V1Service) v1Service));
+          v1Service -> createService((V1Service) v1Service),
+          V1Pod.class,
+          v1Pod -> createPod((V1Pod) v1Pod));
```

Comment:
After making the API members non-static, you need to make this map non-static as well. You can change this to a member and initialize it in the constructor.